src/vm/arm64/cgencpu.h

```diff
@@ -191,9 +191,10 @@ inline BOOL ClrFlushInstructionCache(LPCVOID pCodeAddr, size_t sizeOfCode)
 }
 
 //------------------------------------------------------------------------
-inline void emitJump(UINT32* pCode, LPVOID target)
+inline void emitJump(LPBYTE pBuffer, LPVOID target)
 {
     LIMITED_METHOD_CONTRACT;
+    UINT32* pCode = (UINT32*)pBuffer;
```

Comment:
Never mind - expanding the unmodified code, I see that  we are operating 32bit at a time to write the load and jump.